src/components/input/bl-input.css

```diff
@@ -214,6 +214,10 @@ bl-icon[name='eye_on'] {
   padding: 0;
 }
 
+:host(:not([label-fixed])) :where( .input-wrapper:has(input:autofill) .wrapper:focus-within, .wrapper.has-value) label {
+  background: linear-gradient(to top, var(--autofill-bg-color) 50%, transparent 50%);
+}
```

Comment:
What are we doing here?

Reply:
It adds background to bottom half of the input label to prevent displaying white background shown here:
<img src="https://user-images.githubusercontent.com/47941171/222365507-54a62196-32d0-432c-88a2-73c06222f530.png" />

Makes it:
<img src="https://user-images.githubusercontent.com/47941171/222365508-2314cf1c-f1fd-47b5-b9e9-f636a0c582e1.png" />

Reply:
I'm not sure if we need this. Because by design it has white background in disabled state as well. Maybe @buseselvi can comment about it.

Reply:
yes, white background should stay. we dont need the other one 🙏🏻 @muratcorlu @AykutSarac

Reply:
Honestly it looks very odd and disturbing as an user, but if you believe it effects readability I can remove. When I checked MUI they also fill the background.

<img width="627" alt="Screenshot 2023-03-02 at 11 51 24" src="https://user-images.githubusercontent.com/47941171/222378934-ab3ddf22-e58e-4f46-a27d-870e47af6669.png">

Reply:
I noticed that input doesn't have a background and rest of the components probably too, I think this should be handled with the same separate work

<img width="471" alt="image" src="https://user-images.githubusercontent.com/47941171/222410636-b7398afe-2296-4fd2-913c-47339bc07967.png">

Reply:
I created a separate issue for this #462 If labels will have a change to be transparent, we can also consider it in a separate task. @buseselvi